docker/post_github_comment.py

```diff
@@ -46,62 +47,52 @@ def get_docker_image_size(docker_image, is_contribution: bool = False) -> str:
+    Helper function to convert the Docker image to a valid path.
+    For example, `devdemisto/bottle2:1.0.0.89478.tar.gz` -> `devdemisto_bottle2:1.0.0.89478.tar.gz`.
 
     Arguments:
     - `docker_image` (``str``): The docker image filename.
 
     Returns:
-    - `Path` of the fixed Docker image TAR.
+    - `str` of the fixed Docker image TAR.
     """
+    return f"{docker_image.replace('/', '_')}.tar.gz"
 
-    circleci_root_path = Path(os.environ.get("CIRCLE_WORKING_DIRECTORY", CIRCLECI_DEFAULT_WORKSPACE_DIR))
-    artifacts_path = circleci_root_path / "artifacts"
 
-    return artifacts_path / f"{docker_image.replace('/', '_')}.tar.gz"
+def get_post_url() -> str | None:
+    if os.getenv('PR_NUM'):
+        pr_num = os.getenv('PR_NUM')
+        print("PR number found from environment: " + pr_num)
+        return f'https://api.github.com/repos/demisto/dockerfiles/issues/{pr_num}/comments'
+
+    # try to get from comment
+    last_comment = subprocess.check_output(["git", "log", "-1", "--pretty=%B"], text=True)
+    m = re.search(r"#(\d+)", last_comment, re.MULTILINE)
+    if not m:
+        print("No issue id found in last commit comment. Ignoring: \n------\n{}\n-------".format(last_comment))
+        return None
+    issue_id = m.group(1)
+    print("Issue id found from last commit comment: " + issue_id)
+    post_url = "https://api.github.com/repos/demisto/dockerfiles/issues/{}/comments".format(issue_id)
+    return post_url
 
 
 def main():
     desc = """Post a message to github about the created image. Relies on environment variables:
-GITHUB_KEY: api key of user to use for posting
-CIRCLE_PULL_REQUEST: pull request url to use to get the pull id. Such as: https://github.com/demisto/dockerfiles/pull/9
-if CIRCLE_PULL_REQUEST will try to get issue id from last commit comment
+XSOAR_BOT_GITHUB_TOKEN: api key of user to use for posting
+PR_NUM: the PR number to post to. If not set, will try to get from last commit message.
     """
     parser = argparse.ArgumentParser(description=desc,
                                      formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument("docker_image", help="The docker image with tag version to use. For example: devdemisto/python3:1.5.0.27")
     parser.add_argument("--is_contribution", help="Whether the PR is a contribution or not", action="store_true", default=False)
-    parser.add_argument(
-        "-j",
-        "--job-id",
-        help="The CircleCI workflow job ID. Default is the environmental variable CIRCLE_WORKFLOW_JOB_ID",
-        default=os.environ.get('CIRCLE_WORKFLOW_JOB_ID', ""),
-        required=False,
-        dest="job_id"
-    )
-
     args = parser.parse_args()
-    if not os.environ.get('GITHUB_KEY'):
-        print("No github key set. Will not post a message!")
+    print("Posting to github about image: " + args.docker_image)
+    post_url = get_post_url()
+    if not post_url:
         return
```

Comment:
Is it logged anywhere? If not, then:
```suggestion
        return
    print('Found PR Comments URL: {post_url}')
```